Bitrise/testing_bitrise.yml

```diff
@@ -17,6 +17,11 @@ workflows:
                 echo 'Cancelling CI run since the PR is in draft'
                 exit 1
             fi
+
+            if [[ "$BITRISE_GIT_BRANCH" == *"smartling-content"* ]]; then
```

Comment:
Do we have any "say" on the branch naming from Smartling? Wondering if we can make this more generic, as to not break this if we were to change the localization provider.

Reply:
I don't think we have that flexibility right now. Either way, chances are so low we're changing localization provider that I don't think it's needed to optimise for that